src/services/khbooks/payment-service.ts

```diff
@@ -506,11 +506,18 @@ export const sendPayment = async (id: number) => {
     companies: payment.companies,
   })
 
+  const companyName =
+    payment.companies?.name !== undefined &&
+    payment.companies?.name !== null &&
+    payment.companies?.name.length > 0
+      ? ` from ${payment.companies.name}`
+      : ""
+
   await sendMail({
     to: toEmails,
     cc: ccEmails,
     bcc: bccEmails,
-    subject: `Payment Receipt from ${payment.companies?.name}`,
+    subject: `Payment Receipt${companyName}`,
```

Comment:
please use the same logic in the invoice ^^

Reply:
^^ Done